src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -372,7 +372,7 @@ public static TNode InsertEnvironmentElement(TNode targetNode)
 #else
             env.AddAttribute("clr-version", Environment.Version.ToString());
 #endif
-#if !PLATFORM_DETECTION
+#if NETSTANDARD1_4 || NETSTANDARD2_0
```

Comment:
I _think_ this preprocessor should stay in sync with whatever's wrapping the OSPlatform class (currently PLATFORM_DETECTION). My _guess_ is that this currently works because of the `Microsoft.Win32.Registry` - but will fail to compile once that's removed. 

OSPlatform also uses Registry lookup (seemingly to determine Windows versions...). I think this is the right solution, it will just need updating if the dependency is removed. Either that, or the introduction of a REGISTRY_LOOKUP pre-processor - although that's possibly overkill! 😄